docs/USAGE.md

```diff
@@ -16,7 +16,7 @@ java -jar gtfs-validator-v2.0.jar --input relative/path/to/dataset.zip --output
 ...which will:
  1. Search for a GTFS dataset located at `relative/path/to/dataset.zip`
  1. Validate the GTFS data and output the results to the directory located at `relative/output/path`. 
- 1. Export the validation report to `JSON` by default. This folder will contain a single `.json` file with information related to the validation process. The validation report will be named as `report.json`. 
+ 1. Export the validation and system errors reports to `JSON` by default. This folder will contain a single `.json` file with information related to the validation process. The validation report will be named as `report.json` and the system errors report can be found under the name of `system_errors.json`. 
```

Comment:
Isn't there 2 files now? If yes, the second sentence still mention a single .json file.